{"name": "test_update_pizza_quantity_in_cart", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"d3d7abdd787f4b763445dc8d023a457a\")>\n\n    def test_update_pizza_quantity_in_cart(driver):\n        main_page = MainPage(driver)\n        main_page.open()\n    \n        # Добавим пиццу\n>       pizza_name = main_page.add_pizza_to_cart_by_index(0)\n\npizzeria_tests\\tests\\test_main_flow.py:89: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.main_page.MainPage object at 0x000001384FFD3410>, index = 0\n\n    def add_pizza_to_cart_by_index(self, index):\n        cards = self.driver.find_elements(*self.pizza_cards)\n>       pizza = cards[index]\nE       IndexError: list index out of range\n\npizzeria_tests\\pages\\main_page.py:42: IndexError"}, "start": 1744313123665, "stop": 1744313124239, "uuid": "82bb0835-ea90-4978-aa32-2bd892c638d9", "historyId": "b2fb447100d376027464854cae4f8174", "testCaseId": "b2fb447100d376027464854cae4f8174", "fullName": "pizzeria_tests.tests.test_main_flow#test_update_pizza_quantity_in_cart", "labels": [{"name": "parentSuite", "value": "pizzeria_tests.tests"}, {"name": "suite", "value": "test_main_flow"}, {"name": "host", "value": "Ittekimasu"}, {"name": "thread", "value": "10928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pizzeria_tests.tests.test_main_flow"}]}